demos/demo-minimal-js/index.js

```diff
@@ -142,7 +142,12 @@ export function completeCall() {
     hideWidget: false,
   });
   disableButtons([COMPLETE_CALL]);
-  enableButtons([OUTGOING_CALL, INCOMING_CALL]);
+  const buttonIds = [OUTGOING_CALL];
+  const userAvailableButton = document.querySelector(`#${USER_AVAILABLE}`);
+  if (userAvailableButton.getAttribute("disabled") === "true") {
+    buttonIds.push(INCOMING_CALL);
+  }
```

Comment:
This works! However I'm thinking if we should have a `isUserAvailable` boolean state where we toggle upon `userAvailable()` and `userUnavailable()` instead of making the disabled attribute check for reusability and since our call methods are more declarative functions.

Reply:
Change made. However, there's another edge case where you can toggle between user available and user unavailable during a call and it will enable the incoming call button again. I'm starting to think that we should simply disable the availability buttons during a call, and leave the complex logic to demo-react-ts.
![2023-06-29 16 03 22](https://github.com/HubSpot/calling-extensions-sdk/assets/11711699/33ac4d69-13af-4352-8e39-413d893559d1)

Reply:
Yeah we could do that. How shall we show the user availability state while both buttons are disabled?

Reply:
We could show a toggle element instead of two buttons, but do we need to? I'm not sure we care too much about availability while in a call. We'd also need to change the log in/out buttons for consistency. This is the experience with the buttons disabled during the call: https://github.hubspot.com/calling-extensions-sdk/pr-preview/pr-138/demo-minimal-js.html. What are your thoughts?

Reply:
I agree and I think this experience works for the minimal version!